packages/to-markdown/fixtures/details-private/EXPECTED.md

```diff
@@ -0,0 +1,148 @@
+# `./fixtures/-TEST/package/my-element.js`:
+
+## class: `SuperClass`
+
+### Superclass
+
+| Name       | Module | Package     |
+| ---------- | ------ | ----------- |
+| LitElement |        | lit-element |
+
+### Methods
+
+| Name             | Privacy | Description | Parameters | Return | Inherited From |
+| ---------------- | ------- | ----------- | ---------- | ------ | -------------- |
+| superClassMethod | public  |             |            |        |                |
+
+### Events
+
+| Name         | Type               | Description    | Inherited From |
+| ------------ | ------------------ | -------------- | -------------- |
+| custom-event | `SuperCustomEvent` | this is custom |                |
```

Comment:
does it handle union types correctly? I saw some handling for Parameters, but I didnt see anything for types yet

Reply:
fixed now now. handled them just fine as types, but I actually had to _remove_ some code which double-escaped pipes in param types

Reply:
I might simplify the test cases. if it's g2g, approve and i'll merge when ready